android/src/main/kotlin/com/wire/kalium/presentation/MainActivity.kt

```diff
@@ -39,14 +39,15 @@ class MainActivity : ComponentActivity() {
     }
 
     private fun loginAndFetchConversationList(coreLogic: CoreLogic) = lifecycleScope.launchWhenCreated {
-        login(coreLogic, serverConfig)?.let {
-            val session = coreLogic.getSessionScope(it.tokens.userId)
+        login(coreLogic, serverConfig).let {
+            val session = coreLogic.getSessionScope(it.session.userId)
```

Comment:
I'm just being nitpicky, but I feel like the name 'authSession.session' is wrong 😅
maybe we can revert it back to token